addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -231,6 +231,10 @@ function factory(dependencies) {
                 return;
             }
 
+            if (!channel.isChatChannel) {
+                return;
+            }
```

Comment:
This is returning too early. The update of the counter (last line of method) must still be executed in this case.

Should just prevent `_notifyNewChannelMessageWhileOutOfFocus`, similar to condition with `isOdooFocused` below.

Reply:
I don't understand the role of the focus here.
We don't want notification from a channel at all. Maybe this isn't the best way to resolve this issue.

Reply:
If the Odoo window is currently focused, we don't want to send a native notification. Those are only useful if the user is looking elsewhere, to bring attention to Odoo.

But if Odoo is already "focused", we just want to send a "normal" notification inside Odoo itself (with `notify` or similar).

You don't really need to worry about any of this for this fix though, you should just ensure the method is not called for channels, and my only suggestion is to move your condition to strictly restrict this method call, and not the code that come afterwards in the handler.